engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -1183,7 +1215,11 @@ public void removeContainerAgent(String containerId)
     if (containerAgent != null) {
       // record operator stop for this container
       for (PTOperator oper : containerAgent.container.getOperators()) {
-        StramEvent ev = new StramEvent.StopOperatorEvent(oper.getName(), oper.getId(), containerId);
+        long groupId = 0;
+        if (redeployRequests.get(containerId) != null) {
+          groupId = redeployRequests.get(containerId).getGroupId();
+        }
+        StramEvent ev = new StramEvent.StopOperatorEvent(oper.getName(), oper.getId(), containerId, groupId);
```

Comment:
Not really, I need this container id in "assignContainer", when assignContainer is called for newly allocated container is added to affected containers list and hence the "container start event" for newly added container will be raised.